src/test/cpp/compare/out/winrt/Component.Async.h

```diff
@@ -9,25 +9,25 @@ namespace winrt::impl
     {
         void* operation;
         check_hresult(WINRT_SHIM(Component::Async::IClassStatics)->Action(&operation));
-        return { take_ownership_from_abi, operation };
+        return { operation, take_ownership_from_abi };
```

Comment:
I like the trailing tag better

Reply:
Yep. I also realized that `hresult_error` already has an attach semantic and its trailing, so this is more consistent.